themes/grav/templates/forms/fields/selectize/selectize.html.twig

```diff
@@ -2,7 +2,7 @@
 
 {% block global_attributes %}
     {% if field.selectize is defined %}
-        {% set fieldSelectize = field.selectize|merge({'create': true}) %}
+        {% set fieldSelectize = field.selectize|merge({'create': field.selectize.create ?? true}) %}
```

Comment:
How about:
```twig
{% set fieldSelectize = {create: field.selectize.create ?? true}|merge(field.selectize ?? {}) %}
```

Reply:
Indeed, that works as well. Pushed to branch.